ui/src/app/components/session-listing/session-listing.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="sessions-wrapper">
+  <h3 class="conv-history">Conversion history</h3>
+  <div class="summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</div>
+  <br />
+  <div class="example-container mat-elevation-z3" *ngIf="dataSource.length > 0">
+    <table mat-table [dataSource]="dataSource">
+      <ng-container matColumnDef="SessionName">
+        <th mat-header-cell *matHeaderCellDef>Session Name</th>
+        <td mat-cell *matCellDef="let element">{{ element.SessionName }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="EditorName">
+        <th mat-header-cell *matHeaderCellDef>Editor Name</th>
+        <td mat-cell *matCellDef="let element">{{ element.EditorName }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="DatabaseType">
+        <th mat-header-cell *matHeaderCellDef>Database Type</th>
+        <td mat-cell *matCellDef="let element">{{ element.DatabaseType }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="DatabaseName">
+        <th mat-header-cell *matHeaderCellDef>Database Name</th>
+        <td mat-cell *matCellDef="let element">{{ element.DatabaseName }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="Notes">
+        <th mat-header-cell *matHeaderCellDef>Notes</th>
+        <td mat-cell *matCellDef="let element; let i = index">
+          <div *ngIf="element.Notes">
+            <button class="notes-toggle-button" (click)="notesToggle[i] = !notesToggle[i]">
+              ...
+            </button>
+            <div *ngIf="notesToggle[i]; else short">
+              <p *ngFor="let note of element.Notes">{{ note }}</p>
+            </div>
+            <ng-template #short>
+              <p>{{ element.Notes?.[0].substring(0,10)+ '...' }}</p>
+            </ng-template>
+          </div>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="CreatedOn">
+        <th mat-header-cell *matHeaderCellDef>Created On</th>
```

Comment:
Since its a timestamp rename it to CreatedAt

Reply:
Addressed in the PR 287. Please note that we have updated the column and field name also.